packages/api/src/Lesson/Lesson.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+
+import { Lesson } from './Lesson.entity';
+
+@Injectable()
+export class LessonService {
+  constructor(
+    @InjectRepository(Lesson) private readonly lessonRepository: Repository<Lesson>
+  ) {}
+
+  async findById(id: string): Promise<Lesson> {
+    const lesson = await this.lessonRepository.findOne({
+      where: { id },
+      relations: [
+        'module',
+        'module.path',
+        'module.path.character',
+        'module.previous',
+        'storySection',
+        'storySection.concept'
```

Comment:
I'm wondering if we need to query ALL the relations here. Not a "wrong thing" per se, just curious if it's the most efficient way